src/plotting.jl

```diff
@@ -0,0 +1,23 @@
+#==
+Plot ZonedDateTime, on x-axis.
+We convert it to DateTimes, in the local timezone,
+and we list that timezone in the x-axis label.
+
+This is just one of many options for how to display this info.
+See this PR: https://github.com/JuliaTime/TimeZones.jl/pull/251#issuecomment-603806198
+for details on the options and their tradeoffs.
+==#
+@recipe function f(xs::AbstractVector{<:ZonedDateTime}, ys)
+    # if no zoned date times nothing to do but return empty list
+    isempty(xs) && return xs
+    tz = timezone(first(xs))  # convert all to same timezone as first one
+    new_xs = DateTime.(astimezone.(xs, tz), Local)
+
+    # xguide is the proper name for xlabel
+    label = get(plotattributes, :xguide, "")
+    if !isempty(label)
+        label *= " "  # leave space between original label and the timezone info
+    end
+    xguide := label *= "(timezone: $tz)"
```

Comment:
From our talk I was thinking we should do something like:
```julia
    if isempty(label)
        label = "Time zone: $tz"
    else
        label *= " ($tz)"
    end
    xguide := label

Reply:
Ok, sounds good to me